libs/ui-lib/lib/ocm/components/clusterConfiguration/manifestsConfiguration/components/propTypes.ts

```diff
@@ -17,7 +17,13 @@ export type CustomManifestsFormProps = {
   showEmptyValues: boolean;
   getInitialValues(customManifests: ListManifestsExtended): ManifestFormData;
   getEmptyValues(): ManifestFormData;
-  validationSchema: Yup.Schema<ManifestFormData>;
+  validationSchema: Yup.ObjectSchema<
+    {
+      manifests: object[];
+    } & {
+      manifests: object[];
+    }
```

Comment:
This type of declaration doesn't make sense, the intersection of a type with the same type is idempotent. [PTAL here.](https://www.typescriptlang.org/play?#code/C4TwDgpgBAyg9gWwgFQBYEsB2BzKBeKAbyjACc4wAuKAV0wGtM4B3TAbQF0oBfAKF9CQoAQSbAIpNFlwF4SKTigAyWIhQZFKueunLV8jbgD0R5QDoLUdMCsBnKLeDoANs6jBU0WwEMkUb-baCtgWZvwmUADGcJiOUABu3qQATNSicOKShvhQADykEAhw8dAe0NEISJg23pgAJlAFYM7ekaUY9ki2PtjQzNao-gnezugNic40EAB8vBHRsTaJpACM1EHZBPmFxe3liFU19Y0Qza3t6J0Q3d69UP0eQxNjw5MzQA)